src/openms/source/ANALYSIS/OPENSWATH/MRMDecoy.cpp

```diff
@@ -324,6 +322,40 @@ namespace OpenMS
     return MRMDecoy::reversePeptide(peptide, false, false);
   }
 
+
+  void switchKR(OpenMS::TargetedExperiment::Peptide& peptide)
+  {
+    static std::string aa[] =
+    {
+      "A", "N", "D", "C", "E", "Q", "G", "H", "I", "L", "M", "F", "S", "T", "W",
+      "Y", "V"
+    };
+    int aa_size = 17;
+
+    static boost::mt19937 generator(42);
```

Comment:
Should a seed similar as in shufflePeptide() be used?

Reply:
only if we want to actually test this part, right now all peptides in the test have K/R at the end